crates/snapbox/src/lib.rs

```diff
@@ -114,6 +114,7 @@ pub use action::DEFAULT_ACTION_ENV;
 pub use assert::Assert;
 pub use data::Data;
 pub use data::DataFormat;
+pub use data::{Normalize, NormalizeMatches, NormalizeNewlines, NormalizePaths};
```

Comment:
side note: Hmm, one advantage of an `enum` is its a bit less of a mouthful everywhere.

Reply:
I didn't change to an enum but if it seems like a better fit I can look certainly try to implement it

Reply:
Can't remember what trade offs we considered.  I marked this as a side note because I wanted to be clear this wasn't me prescriptively saying we should switch.  I leave it to you who has been working on this and probably thought through the different angles.